apps/native-component-list/src/screens/ExpoApisScreen.tsx

```diff
@@ -1,20 +1,8 @@
-import { Platform } from 'expo-modules-core';
-import * as Notifications from 'expo-notifications';
 import React from 'react';
 
 import ComponentListScreen from './ComponentListScreen';
 import ExpoAPIIcon from '../components/ExpoAPIIcon';
 
-if (Platform.OS !== 'web') {
-  Notifications.setNotificationHandler({
```

Comment:
I've moved this to the notifications screen because importing `expo-notifications` throws an error as it's not supporting macOS.

Reply:
nit: not sure whether it will cause behavior change, e.g. testing on `GeofencingScreen` will not have notification handler setup. alternatively, we can use `optionalRequire()` or try-catch require here.

Reply:
Ah, you're right. I moved it back to `ExpoApisScreen` and wrapped in try-catch 😉